include/postprocessors/MultiPeriodAverager.h

```diff
@@ -29,16 +29,16 @@ class MultiPeriodAverager : public GeneralPostprocessor
   Real _value;
   /// Where we are going to be storing the next average while we calculate it
   Real _temp_value;
-  /// The number of periods that have passed
-  Real _period_count;
   /// The time when the next period starts
   Real _next_period_start;
-  /// The counter for how many periods have passed since we last updated
-  Real _cyclic_period_count;
   /// inverse of the frequency
   const Real _period;
-  /// the number of periods over which we are averaging
-  Real _num_periods;
   /// The previous post process value of the post processor we are averaging over several periods
   const PostprocessorValue & _pps_value_old;
+  /// The number of periods that have passed
+  uint _period_count;
+  /// The counter for how many periods have passed since we last updated
+  uint _cyclic_period_count;
+  /// the number of periods over which we are averaging
+  uint _num_periods;
```

Comment:
```suggestion
  unsigned int _period_count;
  /// The counter for how many periods have passed since we last updated
  unsigned int _cyclic_period_count;
  /// the number of periods over which we are averaging
  unsigned int _num_periods;
```

Reply:
`unsigned int` is a standard keyword whereas `uint` is typically a `typedef`